web-registry/src/generated/json-ld/project-metadata.ts

```diff
@@ -0,0 +1,94 @@
+import { URL, TypeValue } from '../../types/rdf';
+// import { GeoJSON } from 'geojson';
+import { GeocodeFeature } from '@mapbox/mapbox-sdk/services/geocoding';
+
+export interface ProjectMetadataLD {
+  '@context': Context;
+  '@type': string;
+  'schema:name': string;
+  'schema:image': URL;
+  'schema:description': string;
+  'regen:projectType': string;
+  'regen:projectActivity': ProjectActivity;
+  'regen:projectDeveloper': ProjectStakeholder;
+  'regen:landSteward': ProjectStakeholder;
+  'regen:landOwner': ProjectStakeholder;
+  'regen:projectOriginator': ProjectStakeholder;
+  'regen:offsetGenerationMethod': string;
+  'regen:projectSize': ProjectSize;
+  'regen:projectStartDate': TypeValue;
+  'regen:projectEndDate': TypeValue;
+  'schema:location': GeocodeFeature;
+
+  'regen:boundaries': TypeValue;
+  'regen:creditClass': CreditClass;
+
+  'regen:landManagementActions': LandManagementActions;
+  'regen:galleryPhotos': URLList;
+  'regen:previewPhoto': URL;
+  'regen:videoURL': URL;
+  'schema:creditText': string;
+  'regen:glanceText': TypeValue[];
+  'regen:landStory': string;
```

Comment:
one thing i wanted to point out is that @S4mmyb removed the `landStory` field from the C01 project metadata in this recent PR: https://github.com/regen-network/regen-registry-standards/pull/14/files#diff-163f946557d68f465d976415f679f8dc44aeafccb37dca8bb2c8beea737c1446L34

just wanted to make note of that since i see a reference to it here. maybe that's something we need to add back?

Reply:
I renamed `landStory` with `description` because I thought `description` might make a little bit more sense when we think about describing a variety of credit classes. `landStory` definitely works, but we also might have credit classes with pretty light project pages without much of a story

Reply:
gotcha, makes sense. i'm also just seeing some of the other related convo in https://github.com/regen-network/regen-registry-ops/pull/4

Reply:
Oh yeah. I am fine with calling it `description`.  `landStory` is used in Grasslands projects, but I can have the UI show whichever is available.

Reply:
However, I will leave `landStory` in this particular interface, @wgwz because of the mentioned Grasslands projects. This interface covers all possible project fields, and the VCS interface extends off of it. I thought about making a simpler `base-project` interface, and then extending a `marketing-project` interface off it, but this seemed simple enough for now. I will move `description` to the VCS interface.